### PR TITLE
Fix CI tests

### PR DESCRIPTION
**Pull request by SchSeba**
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/ovs-cni/76)

### CODE REVIEW

File: tests/utils.go [hunk 1]
```diff
@@ -33,6 +34,9 @@ func RunOnNode(node string, command string) (string, error) {
 	}
 
 	out, err := exec.Command("docker", "exec", provider+"-"+node, "ssh.sh", command).CombinedOutput()
+	if err != nil {
+		panic(fmt.Sprintf("failed to run docker exec command %s %s %s %s %s","docker", "exec", provider+"-"+node, "ssh.sh", command))
```
Comment (fedepaol) [inline tests/utils.go]: Just out of curiosity, is there any particular reason on why not calling `fmt.Fatalf` ?

Comment (SchSeba) [reply tests/utils.go]: nope it's just a POC I need to find the issue

**Comment by fedepaol** [comment]
Thanks. Asked because I thought it was something related to recovering from
panics in tests.

**Comment by SchSeba** [comment]
ci test please

File: tests/utils.go (hunk 1 above)
Comment (SchSeba) [reply tests/utils.go]: btw I can't find the `fmt.Fatalf` function

**Pull request merged**